database/create_tbl.py

```diff
@@ -0,0 +1,26 @@
+from time import sleep
+
+import pymysql
+from database.connection import db_connect
+
+def db_createtbl(db):
+    cursor=db.cursor()
+
+
+    studentRecord = """CREATE TABLE tbl_employee1(  
+                   ID int AUTO_INCREMENT PRIMARY KEY,
+                   NAME  VARCHAR(20) NOT NULL,  
+                   Dept VARCHAR(50),  
+                   status VARCHAR(10))"""
+
+
+    validate_create=cursor.execute(studentRecord)
+    db.commit()
+
+    if validate_create==0 or validate_create==None :
```

Comment:
Create a separate function for this validation in the same file. for example: def validate_create_table():
This will improve the readability of the code. It will be easy to understand where we are creating a table and where we are doing table creation validation. You can call validate function from this function only.